src/AeroSharp/DataAccess/MapConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Aerospike.Client;
+using System.Diagnostics.CodeAnalysis;
+
+namespace AeroSharp.DataAccess;
+
+/// <summary>
+///     Configuration that will help build a MapPolicy for Aerospike Map operations.
+/// </summary>
+/// <remarks>
+/// See the <see href="https://www.aerospike.com/apidocs/csharp/html/T_Aerospike_Client_MapWriteFlags.html">documentation</see>
+/// for more information on map write configuration flags.
+/// </remarks>
+[ExcludeFromCodeCoverage]
+public sealed class MapConfiguration
+{
+    public MapConfiguration()
+    {
+        AllowPartial = false;
+        NoFail = false;
+        CreateOnly = false;
+        UpdateOnly = false;
+        Order = MapOrder.UNORDERED;
```

Comment:
Quick thought here, is it worth either setting the default of ordering to "K-Ordered" instead?  While it's not significant necessarily for this PR (and users should know what they're doing when creating the configs), the main operation of get_by_key is O(log N) in the worst case when key ordered as opposed to O(N) for unordered.

Ref: https://docs.aerospike.com/server/guide/data-types/cdt-map-performance

Reply:
Realizing now that the Aerospike client defaults to unordered as well, so no worries here!

Reply:
Sounds good. Thanks for double checking!